integration-tests/goss/windows/tests/command.goss.yaml

```diff
@@ -7,3 +7,17 @@ command:
     - hello world
     stderr: []
     timeout: 10000
+  wrap a powershell - expect 1:
+    exec: powershell -noprofile -noninteractive -command (get-itemproperty -path 'HKLM:/SYSTEM/CurrentControlSet/Control/Lsa/').restrictanonymous
```

Comment:
looks like this command fails:

```
wrap a powershell - expect 1: stdout: patterns not found: [1]
```

Are both of these examples needed?

Reply:
No, definitely not; I added both because I have no idea what the expected value should be within Travis. Adding both was reconnaissance.

Reply:
makes sense :-)